### DISCUSSION TITLE
5기 2주차 회고

### DISCUSSION THREAD

**Discussion by imakerjun** [next-step/blog#247] [회고]
```
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개

## 💡 인사이트 있었던 일

## 🤔 아쉬움이 남은 일

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
```

**Comment by oen142** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
[AWS 구디 모임 발표 후기](https://wani-coding.tistory.com/157)

- 발표를 했었는데 , 생각보다 얻은점이 많아서 의미있는 발표였다.

## 💡 인사이트 있었던 일

- 테스트에 대해서 고민이 있었는데 , 개발자분들과 의견을 나누면서 예시로된 코드를 보면서
많이 배우고 있는점이 인사이트 있었습니다.

## 🤔 아쉬움이 남은 일

- 다른분의 코드를 보면서 많이 배우긴 하는데, 직접 따라 구현해보고 인지하기 까지 시간이 오래걸린다.
이렇게 공부하는게 맞을까?

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)

스프링 업데이트 강의 수강 하기
쉘 스크립트 심화강의 수강하기

**Comment by lns13301** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
자바의 객체 지향 프로그래밍 학습 프로필

## 💡 인사이트 있었던 일
알고 있다고 생각하는 것이 사실은 제대로 알고있지 않다는 것을 깨달았습니다.

## 🤔 아쉬움이 남은 일
앞으로 학습할 때는 학습과 더불어 비슷한 개념과 비교했을 때 차이점을 명확하게 인지하고 누군가에게 설명할 수 있도록 연습 해야겠습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
다음주는 스프링에 대한 본격적인 학습을 하고 해당 학습에 대한 회고 혹은 면접을 보고난 후의 회고를 써볼 계획입니다.

**Comment by pkeugine** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
책을 읽고 내용 정리하기 - 웹과 네트워크의 기본에 대해 알아보자

## 💡 인사이트 있었던 일
다른 분들의 코멘트를 읽고 굉장히 기쁜 마음으로 다음 글을 써야겠다는 생각이 들었습니다.
그리고 다들 정말 글을 잘 쓰시네요! 보고 분발해야겠어요 :)
다 좋은 내용의 글이라 읽기만해도 도움이 많이 되는 것 같습니다.

## 🤔 아쉬움이 남은 일
이번 주에 제가 쓰고자 했던 글을 쓰지 못했습니다. 다음부터는 미리 글을 완성하고 시간 관리를 잘 하기 위해 노력할 예정입니다.

글을 쓸 때 쉽고 빠른, 인스턴스식의 글을 쓸지 아니면 깊고 생각을 많이 한 저만의 글을 쓸지 항상 고민하는데, 후자의 경우는
완성을 못할 때가 정말 많은 것 같네요... 아예 글 쓰는 시간을 한정시켜 놓고 쓸까 생각중...!

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
Spring vs Spring Boot 에 대한 내용을 완성할 예정입니다.

**Comment by imakerjun** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
- 파병일지 중 요즘 핫한 수에즈 운하에서 경험한 내용을 작성했습니다 
- 그 때 온몸으로 꿈꾸던 모습을 다시 떠올릴 수 있어서 스스로 살아있다는 생각이 다시 들 수 있었습니다

## 💡 인사이트 있었던 일
- [ac2](https://ac2.kr/) 교육과정을 듣는데, 책을 전문성 있게 읽는 방법을 알고 무릎을 탁 치게 되었다. 
  - 책읽기에 대한 연구 - SQ3R (Survey, Question, Read, Recite, Review)
  - 타입스크립트 책에 적용해서 사이클을 돌려보고 있는 중
  - 스벨트도 이 사이클로 돌려보고, 하루짜리 워크샵을 만들어보고 싶다.
- [공감적 경청](https://www.youtube.com/watch?v=9AHv-oNkw6k)의 힘에 감탄하게 됨

## 🤔 아쉬움이 남은 일
- [블랙커피 블로그 플랫폼](https://blackcoffee.blog/)에 아직 일반 사용자를 위한 기능을 추가하지 못한점 😢 

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
- 다음주에는 [블랙커피 블로그 플랫폼](https://blackcoffee.blog/) 구독 기능을 추가하고 자랑하고 싶다

**Comment by joelonsw** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
우테코 레벨 1을 정리하면서, 미션 PR링크와 학습로그를 정리했습니다

## 💡 인사이트 있었던 일
사실 저번주에 모의면접 준비 때문에 TIL로 학습로그를 정리해 두었어요. 
저번 주에 TIL을 활용하여 블로그 글 주제를 잡아보자라고 생각했었는데 그렇게 했네요!
너무 날로 먹은 느낌...?이 드는 포스팅이지만, 레벨1을 이제 진짜 정리하고 다음 스텝으로 넘어갈 수 있을 것 같네요!

## 🤔 아쉬움이 남은 일
사실 저는 블로그 글을 쓰면서 많은 생각의 정리가 일어나곤 했어요. 
단순히 배운 지식의 나열이 아니라, 하나하나 점을 이어나가면서 그림을 완성하는 느낌이 들거든요.
조금 더 정제된 포스팅이였으면 좋겠네요. 

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
음... 요즘 기분이 좋아요
조금 더 개인적인 얘기(?) 생각(?)을 블로그에 올려봐야 겠어요.

**Comment by kelly-choi** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
             
- API를 사용하여 데이터를 가져오는 방법에 대해 알 수 있었습니다.

## 💡 인사이트 있었던 일

- OPEN DART에서 제공하는 데이터를 가지고 업계에서 회사의 재무 현황, 투자 현황, 인사 현황을 비교해 볼 수 있고, 경제 지표 산출도 가능하다는 것을 알 수 있었습니다.
## 🤔 아쉬움이 남은 일

- 코드를 작성할 때 좀 더 간결하고 속도를 개선할 수 있도록 작성하도록 공부를 해야할 필요성이 느껴졌습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)

- 파이썬을 통한 크롤링 학습
- 전자 공시 관련 투자, 인사 현황 데이터 추출해보기

**Comment by catsbi** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
2021년 4월 4주차 회고록입니다. 
지난주(4주차)에 있었던 개인공부, 회사업무로인한 출장, 우아한테크코스 프로 프리코스를 진행하며 느낀 생각, 따로진행하는 오프라인 스터디에 대한 여러 생각을 정리해봤습니다. 

## 💡 인사이트 있었던 일
남해로 1박2일 출장을 다녀왔는데, 눈높이에 맞게끔 발표자료를 준비하고 대본 스크립트를 짜는게 얼마나 중요한지 깨달았습니다. 
비록 발표는 다른 개발자분이 했지만 발표를 듣는 대상자가 50~60대 어머니돌봄사분들이셔서 어플에 대한 설치법이나 사용법에 대한 발표를 무척 보수적이고 방어적으로 받아들이시는 부분에서 많은것을 느꼈습니다.
그리고 반대로 1:1로 직접 설치를 도와드리며 얘기를 하니 바로 이해를 하시는분들이 많은 것을 보면 또 아까와는 반대로 사람들이 방어적인것지 바보가 아니다. 라는 점도 깨달았습니다. 

그리고 몇몇 소수의 노약자 가구에는 직접 찾아가서 앱이 설치된 태블릿을 설치하러 방문을 했는데, 직접 설치를 하면서 느끼는 개선사항들을 많이 발견했는데, 이는 회사에서는 깨달을 수 없던 것이였는데, 실무를 모르고 사무실에서만 일을 하는것에 대한 한계를 깨달았습니다. 

## 🤔 아쉬움이 남은 일
저번주보다는 덜하지만 이번주도 분명 여유시간들이 있었는데 최선을 다해 짜투리시간까지 공부를 하진 못했다는 점이 아쉽고, 
출장 출발 직전 앱에 대해서 마지막 테스트를 진행하지 않아서 당일 영상통화가 안되는 문제를 발견 했다는 점에서 후회가 많습니다. 


## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
프리코스, 인프런 인강, 자바스터디 3개가 동시 진행되는데, 
탄력근무제를 최대한 활용하여 프리코스 2주차 마무리, 인프런 인강 스프링 로드맵 종료, 자바스터디 발표준비 및 라이브코딩 자료 준비를 하고싶습니다.

**Comment by humblEgo** [comment]
# 📅 4월 20~26일의 회고

## 📝 블로그 글 소개
[API 설계] DELETE request 요청/처리/응답에 관한 소소한 고민입니다.
DELETE 메서드 처리를 빠르게 구현하는 과정에서 아직 제대로 숙지하지 못한 부분들이 드러나서 했던 고민들을 적었습니다.

## 💡 인사이트 있었던 일
급한 상황에서도 1) 요구사항을 잘 파악해야겠다 2) 요청/응답을 설계하고 피드백을 받은 후 개발을 시작해야겠다 3) 시간이 끝나갈 수록 침착함을 잃지 말아야겠다는 다짐을 했습니다.

지금까지 학습 내용을 너무 '줄 글'로 길게 정리해왔다는 생각이 듭니다. 
오히려 핵심을 요약한 키워드글과 링크를 함께 적는 방식도 좋겠다는 생각이 듭니다.
잘 쓰려는 생각을 버리고, 필요한 글을 깔끔하게 써봐야겠다 싶습니다.

## 🤔 아쉬움이 남은 일
블로그 스터디에 참여하려고 급하게 글을 작성해서 날림글(?)을 작성하고 말았습니다.. 바쁘더라도 틈틈히 글을 적어야겠습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)

1) 미리 글감 하나를 뽑아서 일주일간 틈틈이 작성하겠습니다.
2) 오픈소스 프로젝트, 스터디 프로젝트와 더불어 취업 준비를 한번에 해야하는 상황이 되었는데, 우선순위를 매겨서 잘 병행하겠습니다.

**Comment by bigsaigon333** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
2월~4월초까지 우아한테크코스에서 VanillaJS를 사용하여 다양한 미션을 수행하였습니다. 
그 중 로또 미션과 관련하여 고민하였던 내용을 간략하게 글로 정리해보았습니다.

## 💡 인사이트 있었던 일
자기 전에 리액트 공식문서 내용을 매일 읽고 있는데  그들이 권장(?)하는 방식을 알아 가고 있어 좋습니다. 
그냥 떠다니는 스택오버플로우 내용 보는것보다는 기본기를 탄탄히 다지는 느낌이라 좋습니다~!

## 🤔 아쉬움이 남은 일
따로 시간을 내서 새로운 주제를 가지고 블로그글을 쓰지 말고, 매일 하는 일 또는 해야만 하는 일 가운데서 자연스럽게 블로그 글을 쓰자고 저번주에 다짐했었습니다.  그런데 뭔가 새로운 것, 관심있었던 것에 대한 내용을 작성하지 못해서 약간 아쉽다? 는 생각이 드네요. 
일상에서 벗어나 완전 새로운 주제를 쓰는 소소한 재미가 있는데 말이죠 ㅎㅎ..

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
데일리 회고를 어제부터 노션을 사용해서 하고 있습니다. 
주말에 노션에 어지러히 쌓여있던 페이지들을 일단 1차로 정리를 했는데, 2차로 나머지도 정리하고 싶습니다. 
블로그 글 쓸 주제도 노션 어딘가 써놨는데, 정리하면서 찾아봐야겠어요!

**Reply by bigsaigon333** [reply]
추후 수정시 인사이트 받았던 내용만 요약해서 본문 처음에 2~3줄로 정리해두자!!

**Comment by LimHanGyeol** [comment]
# 📅 4월 20~26일의 회고
## 📝 블로그 글 소개
JVM와 구조에 대해서 소개하는 글을 작성했습니다.

## 💡 인사이트 있었던 일
예전에 개인적으로 한번 정리했지만 이를 포스팅 목적으로 다시 보니 부족한 부분이 많아 추가적인 공부를 하게 됐습니다. 다시 공부하면서 좀 더 복기 및 자세하게 알게 됐습니다. :)

## 🤔 아쉬움이 남은 일
포스팅을 하려고 하다보니 디테일이 떨어지는건 아닐까... 하고 계속 수정을 하게 됐습니다.
그렇다보니 많은 구조에 비해 정리량이 적어서 이렇게 해도 될까... 하는 아쉬움이 들었습니다.
글을 적을때는 깊이에 대해서도 잘 타협하며 적어야 할 것 같습니다.. !

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
첫 포스팅은 기술 알려주는 목적의 포스팅을 했는데, 다음 포스팅은 회고나 개념적인 내용의 포스팅을 해봐야겠다고 생각들었습니다.
